scripts/testing/.gitignore

```diff
@@ -0,0 +1,4 @@
+nohup.out
```

Comment:
Let's move this nohup out of this directory please and into the `tmp/` directory in the raspberry-noaa-v2 folder. I think this is a better move so we don't get too crazy about temp files that don't matter to the framework being littered all over the place.